tools/deb/test/machine_setup.sh

```diff
@@ -59,6 +73,23 @@ server {
     }
 EOF
 
+(cd productpage; sudo pip install -r requirements.txt)
+(cd ratings; npm install)
+
+systemctl restart nginx
+
 # Start istio
 systemctl start istio
 
+systemctl start mariadb
+
+# Start bookinfo components
+ruby details/details.rb 9080 &
+echo $! > details.pid
+
+# Note that we run productpage on a different port - 9080 is taken
+(cd productpage ; python productpage.py 9081 &)
+echo $! > productpage.pid
+
+(cd ratings ; node ratings.js 9082 &)
```

Comment:
Where is reviews?

Reply:
I would rather leave it in K8S - java and all the other deps.

AFAIK we only want to run one of the components on the VM for the demo, I was not 
sure which, will remove the others.

Reply:
just the mysql. nothing else.

Reply:
Sorry, just realized. We don't want to run the database in a VM, as it doesn't make for a convincing auth/mixer story [DBs have dedicated hardware, access policies embedded in the database engine, and so on]. As a user, I would immediately understand a use case where we run one of the services in the VM (use ratings itself, or perhaps as Kelsey suggested, add a simple Go based binary to the VM - indicating a legacy service). Its relatively easy and straightforward to do this [just ask people to run ratings-v1 in VM and remove the k8s version]. This eliminates all the mongo/mysql trouble as well.